backend/app.py

```
@@ -42,6 +45,23 @@
 LABEL_COLUMN_CANDIDATES = ["Label", "label", "Class", "class", "Target", "target"]
 
+# ---------------------------------------------------------------------------
+# MongoDB init
+# ---------------------------------------------------------------------------
+
+try:
+    init_db()
+    seed_if_empty()
+    MONGO_AVAILABLE = True
+    logger.info("MongoDB connected and initialised")
+except Exception:
```

Ruff (0.16.2)

[warning] Do not catch blind exception: `Exception`

(BLE001)